packages/flutter_test/lib/fix_data/README.md

```diff
@@ -0,0 +1,46 @@
+## Directory contents
+
+The `.yaml` files in these directories are used to
+define the [`dart fix` framework](https://dart.dev/tools/dart-fix) refactorings
+used by the Flutter framework.
+
+The number of fix rules defined in a file should not exceed 50 for better
+maintainability. Searching for `title:` is a given `.yaml` file will account
+for the number of fixes. Splitting out fix rules should be done by class.
+
+When adding a new `.yaml` file, make a copy of `template.yaml`. If the new
+file is not for generic library fixes (`fix_material.yaml`), ensure it is
+enclosed in an appropriate library directory (`fix_data/fix_material`), and
+named after the class. Fix files outside of generic libraries should represent
+individual classes (`fix_data/fix_material/fix_app_bar.yaml`).
```

Comment:
Updated this wording entirely to be based off of files and classes and exclude library level fixes since there's only a single public library in `flutter_test`.